db/models/topics.models.js

```diff
@@ -80,9 +84,19 @@ exports.retrieveAllArticles = (sort_by = "created_at", order = "DESC") => {
     "article_img_url",
   ];
   const validOrders = ["ASC", "DESC"];
+  const validTopics = ["mitch", "cats"];
```

Comment:
we should get the valid Topics back from the database instead of hardcoding the topics in an array.